tornado-unittests/src/main/java/uk/ac/manchester/tornado/unittests/fields/TestFields.java

```diff
@@ -201,41 +201,33 @@ public void testSetNestedArray() {
     }
 
     private static class Foo {
-        //        final IntArray output;
-        //        final IntArray a;
-        //        final IntArray b;
-        final int[] output;
-        final int[] a;
-        final int[] b;
+                final IntArray output;
+                final IntArray a;
+                final IntArray b;
 
         Foo(int elements) {
-            //            output = new IntArray(elements);
-            //            a = new IntArray(elements);
-            //            b = new IntArray(elements);
-            output = new int[elements];
-            a = new int[elements];
-            b = new int[elements];
+                        output = new IntArray(elements);
+                        a = new IntArray(elements);
+                        b = new IntArray(elements);
```

Comment:
can you please check the formatter? the spacing seems odd.

Reply:
I ran `make checkstyle` before committing and it passed

Reply:
My understanding is that `checkstyle` fails, I opened a new PR to fix it. 

I used the `scripts/templates/eclipse-settings/Tornadovm_eclipse_formatter.xml` from the `Adapter from Eclipse Code Formatter` in IntelliJ and it changed those lines.

Reply:
Hm....I ran it again now and I am getting the errors you mentioned in the PR #415. I don't get any errors regarding this unittest, but I just noticed that we have the checkstyle off for all unittests. I removed the extra white space.

Reply:
thanks @mairooni.